pkg/controller/dbsyncer/subscription_db_to_transport_syncer.go

```diff
@@ -0,0 +1,38 @@
+package dbsyncer
+
+import (
+	"fmt"
+	"time"
+
+	appsv1 "github.com/open-cluster-management/multicloud-operators-subscription/pkg/apis/apps/v1"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/bundle"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/db"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/intervalpolicy"
+	"github.com/stolostron/hub-of-hubs-spec-transport-bridge/pkg/transport"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	subscriptionsTableName     = "subscriptions"
+	subscriptionRuleMessageKey = "subscriptions"
```

Comment:
why is it called subscriptionRule?
also, in msg keys we usually use capital letters (mainly for consistency)..
so should be:
```
subscriptionsMsgKey = "Subscriptions"
```